e2e/scripts/pageHelpers.js

```diff
@@ -161,9 +184,10 @@ export const registerShopper = async ({page, userCredentials, isMobile = false})
         .locator("input#password")
         .fill(userCredentials.password);
 
+    const tokenResponsePromise=page.waitForResponse('**/shopper/auth/v1/organizations/**/oauth2/token')
```

Comment:
As a best practice, it is always better to await the network call and assert on the network response rather than waiting for pageLoadState(). If an assert after the await for pageLoadState times out, the lock for pageloadstate will be released while the previous network call is still in progress. This will lead to race conditions where if the network call takes longer than the await timeout, the test flow will incorrectly assume the network called failed.

Reply:
Do we want this in a comment somewhere so we don't forget to do this in future tests?